### PR TITLE
Fix YOLOv8 Chinese introduction

### PR DESCRIPTION
**Pull request by Zengyf-CVer**
## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Simplified code example in Chinese README and updated contributors link.

### 📊 Key Changes
- Removed specific method calls (`train`, `val`, `predict`, `export`) from the Python code example in the Chinese README (README.zh-CN.md).
- Updated the contributors' image link to point to the correct repository (`ultralytics/ultralytics` instead of `ultralytics/yolov5`).

### 🎯 Purpose & Impact
- The simplification of the code example aims to reduce clutter and make the README more approachable, emphasizing model usage rather than detailing methods.
- By fixing the contributors' link, it accurately reflects the community behind the `ultralytics` repository, strengthening community recognition and trust.
- Users may benefit from a clearer, streamlined introduction to the library, and contributors receive proper acknowledgment for their work in the project.

### CODE REVIEW

**Comment by Zengyf-CVer** [comment]
I have read the CLA Document and I sign the CLA

**Comment by Zengyf-CVer** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/ultralytics/assets/blob/main/documents/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I sign the CLA
> 
> You can retrigger this bot by commenting **recheck** in this Pull Request

I have read the CLA Document and I sign the CLA

**Comment by Laughing-q** [comment]
@Zengyf-CVer thanks! BTW the English README.md has updated some links and usage(like this PR [https://github.com/ultralytics/ultralytics/pull/213](https://github.com/ultralytics/ultralytics/pull/213)), would you help sync it to Chinese version in this PR?

**Comment by Zengyf-CVer** [comment]
@Laughing-q I have translated.

File: README.zh-CN.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
   </div>
   <br>
 
-[Ultralytics YOLOv8](https://github.com/ultralytics/ultralytics) 是由 [Ultralytics](https://ultralytics.com) 开发的一个前沿 SOTA 模型。它在以前 YOLO 版本的成功基础上，引入了新的功能和改进，进一步提升了性能和灵活性。YOLOv8 基于快速、准确和易于使用的理念设计，使其成为广泛的物体检测、图像分割和图像分类任务的绝佳选择。
+[Ultralytics YOLOv8](https://github.com/ultralytics/ultralytics) 是由 [Ultralytics](https://ultralytics.com) 开发的一个前沿的SOTA模型。它在以前成功的YOLO版本基础上，引入了新的功能和改进，进一步提升了其性能和灵活性。YOLOv8 基于快速、准确和易于使用的设计理念，使其成为广泛的目标检测、图像分割和图像分类任务的绝佳选择。
```
Comment (RangeKing) [inline README.zh-CN.md:21]: 建议中英文之间保留原有空格，利于阅读
```suggestion
[Ultralytics YOLOv8](https://github.com/ultralytics/ultralytics) 是由 [Ultralytics](https://ultralytics.com) 开发的一个前沿的 SOTA 模型。它在以前成功的 YOLO 版本基础上，引入了新的功能和改进，进一步提升了其性能和灵活性。YOLOv8 基于快速、准确和易于使用的设计理念，使其成为广泛的目标检测、图像分割和图像分类任务的绝佳选择。
```
建议保留中英文之间空格，全文统一

Comment (Zengyf-CVer) [reply README.zh-CN.md]: Thanks for your suggestion, I have modified it.

**Comment by glenn-jocher** [comment]
@Zengyf-CVer thanks for the translation fixes!

**Pull request merged**